components/treeview/drag-drop.md

```diff
@@ -49,13 +79,212 @@ The `OnDrop` event provides an object of type `TreeViewDropEventArgs` to its eve
 | `DestinationIndex` | `string` | The index where the drop will happen in the second component. |
 | `DestinationComponentId` | `string` | The `Id` of the second component in which the drop will happen. |
 
+## DragEnd Event
+
+The `DragEnd` event fires when a drag operation is ended by releasing a mouse button. The event is triggered after the OnDrop Event.
```

Comment:
It's important to explain the difference between the `OnDragEnd` and the `OnDrop` events. @YanushevGeorgi  As we discussed in a private conversation, the `OnDrop` event is triggered when an item is dropped over a Telerik component, such as another TreeView. 

On the other hand, the `OnDragEnd` event is triggered when the drag operation ends. In cases where the item is dropped over a different element ( any HTML element - not a Telerik component), the `OnDragEnd` event arguments will be `null`.

Additionally, the phrase 'by releasing a mouse button' may be better removed, as in the future, we might cancel the drag operation on other events, like clicking the `Esc` key. 

'The `DragEnd` event fires when a drag operation is ended' seems enough.